sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/CredentialEntityLiveTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Administration;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class CredentialEntityLiveTests : MetricsAdvisorLiveTestBase
+    {
+        private const string ClientId = "clientId";
+        private const string ClientSecret = "clientSecret";
+        private const string TenantId = "tenantId";
```

Comment:
does the service does any validation to see that the values are legit?

Reply:
Not during creation. It will provide error messages showing that data ingestion has failed when a user calls the `GetDataFeedIngestionStatuses` API.

Reply:
not much we can do here. just wondering if a user will expect this method to fail if the connection wasn't able to be established.
maybe adding something about it in the docs?